### PR TITLE
test: add terraform config for Cloud Build trigger

### PR DESCRIPTION
**Pull request by tswast**
In response to https://github.com/GoogleCloudPlatform/professional-services/pull/465#issuecomment-622071276

### CODE REVIEW

File: cloudbuild/main.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# limitations under the License.
+
+resource "google_cloudbuild_trigger" "default-push-trigger" {
+    provider = google-beta  # Use Beta version for GitHub support
+    project = var.gcp_project_id
+    github {
+        owner = var.github_owner
+        name = var.github_repo_name
+        push {
+            branch = ".*"
```
Comment (jaketf) [inline cloudbuild/main.tf:22]: could be useful to variable-ize branch.

Comment (jaketf) [reply cloudbuild/main.tf:22]: For example if in a contributor's workflow is on their fork they rage commit / push to a wip branch and then merge to master on their fork when they are ready run CI checks.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,24 @@ To get started contributing:
...
 1. Fork the repo, develop and test your code changes.
+
+   To test your changes before making a pull request, create a Cloud Build
+   trigger for your fork on your own project.
+
+   ```
+   gcloud config set project YOUR-PROJECT
+   cd cloudbuild
+   teraform init
+   terraform apply -var='gcp_project_id=YOUR-PROJECT' -var='github_owner=GITHUB-USER-ID'
```
[the remark this answers is not in the document]
Comment (tswast) [reply CONTRIBUTING.md]: Done in https://github.com/GoogleCloudPlatform/professional-services/pull/467/commits/1344d825e0fc467713a3c1e69d5297685e304e30

File: cloudbuild/main.tf (hunk 1 above)
Comment (jaketf) [reply cloudbuild/main.tf]: bump. thoughts on variablizing this branch pattern?

Comment (tswast) [reply cloudbuild/main.tf]: done in 1344d825e0fc467713a3c1e69d5297685e304e30

**Pull request merged**